.github/workflows/check-for-build-warnings.yml

```diff
@@ -10,8 +12,8 @@ on:
 
```

Comment:
@BillWagner For the manual run to be useful, we'd have to pass in a PR number. Do you want me to change the action to take a PR number as input?

https://github.com/dotnet/docs/runs/2273417489?check_suite_focus=true

Reply:
Ping @IEvangelist who taught me this trick on the manual run.

Reply:
@gewarren yes, you'd have to update the action to take an optional PR number. Then you'd essentially [coalesce on the payload](https://github.com/dotnet/samples/blob/main/.github/actions/status-checker/src/status-checker.ts#L10), as that would be `null` when manually ran.

Reply:
Thanks, I think it's best to just remove the manual trigger for now. We can add it later if we decide it's needed.